terraform-build-user/main.tf

```
@@ -11,4 +11,8 @@ module "iam_user" {
 
   ssm_parameters = ["/windows/commando/administrator/password"]
   user_name      = "build-windows-commando-vm-packer"
+
+  # This image can take a while to build, so we set the max session
+  # duration to 2 hours.
+  ec2amicreate_role_max_session_duration = 2 * 60 * 60
```

We prefer to sort these.

```suggestion
  # This image can take a while to build, so we set the max session
  # duration to 2 hours.
  ec2amicreate_role_max_session_duration = 2 * 60 * 60
  ssm_parameters = ["/windows/commando/administrator/password"]
  user_name      = "build-windows-commando-vm-packer"
```

---

Resolved here https://github.com/cisagov/windows-commando-vm-packer/pull/21/commits/3add97b63d4d48f1aa509a5dcbfce84a4c745e3d